xl/xenops_server_xenlight.cppo.ml

```diff
@@ -2545,7 +2545,7 @@ let with_data ~xs task data write f = match data with
             finally
               (fun () -> f fd)
               (fun () ->
-                 try Fsync.fsync fd;
+                 try Stdext.Unixext.fsync fd;
```

Comment:
@mseri said that we should move to using specific top-level modules of stdext, such as:
```ocaml
Xapi_stdext_unix.Unixext.fsync
```

Reply:
Because then we won't link in the whole of stdext, if I remember correctly.